File: python/mathfun.py
def getHighest(numbers) -> int:
    highest = 0
    for number in numbers:
        if number > highest:
            highest = number
    return highest


def printNumbers(numberList):
    for number in numberList:
        print(number)


printNumbers({4, 5, 6, 7})
print(getHighest({4, 5, 6, 7}))

nums = {4, 5, 6, 7}
printNumbers(nums)

File: csharp/console_fun/src/MathFun.cs
namespace src
{
  public static class MathFun
  {
    public static void PrintNumbers(List<int> numbers)
    {
      foreach (int num in numbers)
      {
        Console.WriteLine(num);
      }
    }

    public static bool IsEven(int number)
    {
      if (number % 2 == 0)
        return true;
      else
        return false;
    }
  }
}
